### PR TITLE
i18n(ko-KR): update `errors/` and `error-reference.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

update `errors/` and `error-reference.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by jsparkdev** [comment]
Once #10166, #10201 are merged, I expect to be able to merge this PR successfully.

**Review by yanthomasdev — APPROVED** [review]
LGTM! I'll merge the umbrella PR once I approve all of the individual PRs.

**Pull request closed without merging**
